pkgs/applications/networking/cluster/atmos/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "atmos";
+  version = "1.4.28";
+
+  src = fetchFromGitHub {
+    owner = "cloudposse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AqW//+x1tKzm5dJirXCLFuDBTk3dHHw7c6vwsVsI37U";
+  };
+
+  vendorSha256 = "sha256-bepv8Ns8vcpLpmO7fsRgGijB9B6kQ5HhxUhIuEpGtOY=";
+
+  ldflags = [ "-X 'github.com/cloudposse/atmos/cmd.Version=${version}'" ];
+
+  doCheck = false;
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+    $out/bin/atmos version | grep "${version}"
+    runHook postInstallCheck
+  '';
+
+  meta = with lib; {
+    homepage = "https://atmos.tools";
+    changelog = "https://github.com/cloudposse/atmos/releases/tag/v${version}";
+    description = "Universal Tool for DevOps and Cloud Automation (works with terraform, helm, helmfile, etc)";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ jk ];
```

```suggestion
    maintainers = with maintainers; [ ];
```

Please don't sign me up as maintainer :joy:

Looks like you'll need to add yourself to the maintainers list. please do this in a separate commit as explained in CONTRIBUTING.md

---

Fixed! I will add a new maintainer in a separate PR then.